### ISSUE TITLE
Unify collection lists

### ISSUE THREAD

**Issue by vikiival** [kodadot/nft-gallery#2817]
### What happened?

I saw that we have two types of components which are may doing the same ?? 

```ts
import CollectionList from '@/components/rmrk/Collection/List/CollectionList.vue'
import Collections from '@/components/rmrk/Gallery/Collections.vue'
```

```vue
<template>
  <CollectionList v-if="type === 'rmrk'" />
  <Collections v-if="type !== 'rmrk'" />
</template>
```

### Please reproduce in steps

~

### Expected Behavior

I've expected to have one reusable component across the app 🙈 

### Screenshots

~

### Payment link for reward

https://kodadot.xyz/transfer/?target=%3CMy_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address%3E

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by prachi00** [comment]
@vikiival can we add priority to this?

**Comment by prachi00** [comment]
@roiLeo it would get merged with #2812

**Comment by roiLeo** [comment]
meanwhile `@/components/rmrk/Gallery/Collections.vue` is not used anymore, IMO we can safely remove it and close this issue
